addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -131,7 +127,7 @@ export class Popup extends Interaction {
         }
 
         if (display === "afterDelay") {
-            this.waitForTimeout(this.showPopup, delay);
+            this.waitForTimeout(this.showPopup.bind(this), delay);
```

Comment:
we should probably change Interaction to autobind functions in this.waitForTimeout and waitFor

Reply:
There's another PR for that :D https://github.com/odoo-dev/odoo/pull/3581